tasks.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import os
+import subprocess
+import sys
+
+from invoke import task
+
+logging.basicConfig(level=logging.INFO,
+                    format='%(levelname)s - %(message)s')
+
+APP_NAME = 'envars'
+PYTHON_VERSION = 3.10
```

Comment:
should be loaded from the PYTHON_VERSION file?

Reply:
it actually falls back to this if a PYTHON_VERSION is not found.   I guess it should always be set